model_components/meas.Position.xml

```diff
@@ -0,0 +1,59 @@
+<INSTANCE dmrole="" dmtype="meas:Position">
+   <!-- Provides a complete description of a measured positional instant." -->
+   <ATTRIBUTE dmrole="meas:Measure.ucd" dmtype="ivoa:string" ref="@@@@@" value=""/>
```

Comment:
The Position measure is constrained to ucd='pos', so this should not be open in this example.
There is the possibility that other tokens could be useful (eg: 'meta.main'), which we decided to wait and see how the use case for this works out (we don't want this ucd field to include modeled features.. it should just specify the type).